### DISCUSSION TITLE
something about integrating scATAC-seq into scRNA-seq

### DISCUSSION THREAD

**Discussion by tianchen2019** [GreenleafLab/ArchR#648] [Questions / Documentation]
I would like to ask why when integrating scATAC-seq into scRNA-seq data, a large number of cells(scATAC-seq) are integrated into the same cell (scRNA-seq).
i have a scATAC-seq of 380000 cells, and only integrated 23000 scRNA-seq cells.

**Comment by rcorces** [comment]
Integration in ArchR is managed through Seurat's infrastructure (`FindTransferAnchors()`). The best matching cell from RNA-space is used for each cell in ATAC-space. Presumably there is a small subset of cells in your RNA-space that are repeatedly serving as the best match for your ATAC cells. This could be data quality or it could be that you need to tweak parameters in `addGeneIntegrationMatrix()`.
https://www.archrproject.com/reference/addGeneIntegrationMatrix.html
